src/archiefbeheercomponent/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -476,90 +550,180 @@ msgstr ""
 "{{ uuid }}, {{ bronorganisatie }} en {{ identificatie }}. Bijvoorbeeld: "
 "https://gemeente.lan/mijnzaken/zaak/{{ uuid }}"
 
-#: archiefvernietigingscomponent/destruction/models.py:393
+#: archiefbeheercomponent/destruction/models.py:416
+msgid "days until reminder"
+msgstr "dagen tot herinnering"
```

Comment:
Typically I don't like these kind of translations. They come from constructions like this: `days + _("days left")` but what you should do is `_("{days} days left").format(days=days)`.

This allows for languages to move the variabele around, for example it could be translated as: "nog {days} dagen resterend"

Translation is correct though :)

Reply:
This is the name of a field on a model, so it is not used in this case for a `days + _("days left")` construction

Reply:
fair enough :)